BasiliskII.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Should be renamed to match the app-id & contributed upstream if possible

Reply:
Can do.